modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -306,6 +308,19 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     ]);
     // #endregion
 
+    // #region reducing keychain data
+
+    userKeychain.encryptedPrv = this.bitgo.encrypt({
```

Comment:
I think it might be better to just return this as a new field.

Reply:
And update `generateKeycard` to use compressed keys if the exist, else fall back to the default encrypted keys.

Reply:
Updating the keys in place like this might have un-intended effects as you cant guarantee this is always to be used for key card generation or related services?

Reply:
i agree adding a new field is much safer, will apply to the PR

Reply:
done